### ISSUE TITLE
Jobs are stuck after a restart

### ISSUE THREAD

**Issue by heurtematte** [eclipse-cbi/jiro#350]
This is a recurring issue where the jenkins build queue is stuck after a reboot.

Workaround: clear the queue with `Jenkins.instance.queue.clear()`

Related to https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4306#note_1754550

**Comment by fredg02** [comment]
This might help: https://plugins.jenkins.io/purge-build-queue-plugin/

**Comment by fredg02** [comment]
Yep, but we would still need to do it manually. Would be nice to allow this action only for project leads.

@mbarbero wdyt about a groovy init script that clears the queue after every Jenkins start?

**Comment by mbarbero** [comment]
> Yep, but we would still need to do it manually. Would be nice to allow this action only for project leads.

agreed.

> @mbarbero wdyt about a groovy init script that clears the queue after every Jenkins start?

it depends :) Do we have cases where a Jenkins restart actually lead to successful jobs restart? If that's the case, then it means that by implementing your plan, you will effectively solve your issue, but also remove the auto-restart feature for others.

**Comment by heurtematte** [comment]
+1 groovy init script 

If it don't work as expected, install `purge-build-queue-plugin for project.

**Comment by lredor** [comment]
On https://ci.eclipse.org/sirius, I observed that some jobs have dissapered (without any information, as if they never existed). For example, a job [sirius.sonar-master](https://ci.eclipse.org/sirius/job/sirius.sonar-master/) has been launched this morning. But it never ended and it is no longer visible. Is there a link with this issue ?
